try_task_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file will be removed, just added to see that the ui-test-x86-nightly task is created successfully